src/Package/Updater.php

```diff
@@ -274,11 +274,12 @@ public function update(IOInterface $io, Config $config, Package $package, VcsRep
     }
 
     /**
-     * @return array with keys:
-     *                    - updated (whether the version was updated or needs to be marked as updated)
-     *                    - id (version id, can be null for newly created versions)
-     *                    - version (normalized version from the composer package)
-     *                    - object (Version instance if it was updated)
+     * @return array{updated: bool, id: int|null, version: string, object: Version|null}
```

Comment:
Maybe better would be `array{updated: true, id: int|null, version: string, object: Version}|array{updated: false, id: int|null, version: string, object: null}`. It's a bit messy with duplication but it would fix the error above I hope.

Reply:
ok, lets see whether this works